hw_twitterapi/twitterapiapp/templates/twitterapiapp/recent_favorites.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
We have a base template and all api calls inherit it. This is done for keeping our project theme the same. Please use inheritance. Refer to other doers of the project.